02/4_multi_dimensional_array/main.c

```diff
@@ -0,0 +1,23 @@
+#include <stdio.h>
+
+int main(void)
+{
+    int grades[2][3] = {

```

Comment:
in c this is where u would use define at the top of a .c file

#define NUM_CLASSES (2)
#define NUM_SUBJECTS (3)

i briefly talked about #define during week 1. and it's probably a good palce to talk about it unofficially again in code slides